examples/virtual-networks/private-link-service/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "resource_group_name" {
+  description = "The name of the resource group the Private Link Service is located in."
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this example should be created."
+}
```

Comment:
Would it be possible to add a newline here?

Reply:
Updated